### DISCUSSION TITLE
Auto position drop tree component based on scroll position

### DISCUSSION THREAD

**Discussion by rajeevprasanna** [dowjones/react-dropdown-tree-select#500] [Q&A]
i am using npm library react-dropdown-tree-select to render tree component.

I want to auto position dropdown down/up based on scroll position on the page.

if this component is placed at the bottom of the page, I expect a dropdown to come on top. can someone suggest a way to accomplish this?

**Comment by mrchief** [comment]
Hey, @rajeevprasanna I think this was answered in an old issue but the simple solution to that would be to:

 - create styles for displaying the dropdown content above the textbox
 - create an HOC that checks for its position on the page and switch the css classes when it's at page bottom.
